.github/workflows/cancel_duplicates.yml

```diff
@@ -1,25 +1,40 @@
 name: Cancel Duplicates
 on:
   workflow_run:
-    workflows: ["CI"]
-    types: ["requested"]
+    workflows:
+      - "Miscellaneous"
```

Comment:
so once merge and will work only with Miscellaneous workflow then we'll add all workflows here right?

Reply:
Only this workflow will trigger the cancel job, but a scheduled cancel job will try to cancel duplicates of all workflows.